pkgs/applications/kde/rocs.nix

```diff
@@ -1,7 +1,7 @@
 {
   mkDerivation, lib,
   extra-cmake-modules, boost,
-  qtbase, qtscript, qtquickcontrols, qtwebkit, qtxmlpatterns, grantlee,
+  qtbase, qtscript, qtquickcontrols, qtxmlpatterns, grantlee,
```

Comment:
Nit: The changes in `kdeApplications: remove qtwebkit dependency` IMO should be split to 3 - each change for each package.